_data/people.yml

```
@@ -244,7 +246,7 @@
   url: /positions
   description: 'Ideal ARFC group members seek to advance nuclear energy challenges through advanced computing, physics-informed modeling, and sophisticated simulation'
   social:
-    - url: mailto:katyhuff@gmail.com
+    - url: mailto:Madicken.munk@gmail.com
```

I'm going to add my illinois here because I use it more for work! 

```suggestion
    - url: mailto:mmunk2@illinois.edu
```